resources/lib/item_functions.py

```diff
@@ -7,20 +7,15 @@
 from datetime import datetime
 
 import xbmc
-import xbmcaddon
 import xbmcgui
 
 from .utils import datetime_from_string, get_art_url, image_url
-from .loghandler import LazyLogger
+from .lazylogger import LazyLogger
 from six import ensure_text
 
 log = LazyLogger(__name__)
 kodi_version = int(xbmc.getInfoLabel('System.BuildVersion')[:2])
```

Comment:
Might as well use your fancy new kodi_version() function here too.

Reply:
completely forgot about that when I hit rebase hell :laughing: